model/simulation.py

```diff
@@ -178,68 +182,63 @@ def refill_amount(self, block_height, expected_new_vaults):
         refill_amount += cf_fee
         return int(refill_amount)
 
-    def _reserve_divergence(self, block_height):
+    def compute_reserve_divergence(self, block_height):
         """Compute how far the vault's reserves have divereged from the current fee reserve per vault.
         Compute the risk status; the total amount (satoshis) below the required reserve among available vaults."""
         vaults = self.wt.list_available_vaults()
-        if vaults != []:
-            divergence = []
-            required_reserve = sum(self.wt.coins_dist_reserve(block_height))
-            for vault in vaults:
-                div = vault.reserve_balance() - required_reserve
-                divergence.append(div)
-            if self.with_divergence:
-                self.divergence.append(
-                    [
-                        block_height,
-                        sum(divergence) / len(vaults),
-                        min(divergence),
-                        max(divergence),
-                    ]
-                )
+        if len(vaults) == 0 or not (self.with_divergence or self.with_risk_status):
+            return
+
+        divergence = []
+        for vault in vaults:
+            div = vault.reserve_balance() - self.required_reserve(block_height)
```

Comment:
This should be
```python3
div = vault.reserve_balance() - (self.required_reserve(block_height)/self.wt.vaults_count())
```

Reply:
i.e. divergence of the vault's balance from the requirement _per vault_

Reply:
This was not introduced here, so the divergence computation was wrong all this time?

Added the fix in a new commit, good find!